src/kernel/mailbox.rs

```diff
@@ -327,13 +327,13 @@ fn handle_evt<A>(
     }
 }
 
-struct Sentinel<Msg: Message> {
+struct Sentinel<'a, Msg: Message> {
     parent: BasicActorRef,
```

Comment:
will these changes affect current public API?

Reply:
No. This is just an internal struct which suspends a mailbox in case the thread is panicking.